### PR TITLE
Dojo 7 roadmap entry

### PR DESCRIPTION
**Pull request by agubler**
## Description

Initial roadmap update for the dojo 7 release, currently only includes the updates from `@dojo/framework`.

### CODE REVIEW

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* next-dojo-io: https://next-dojo-io-rnnae6rcm.now.sh
* zh-cn-next-dojo-io: https://zh-cn-next-dojo-io-9hslut250.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* next-dojo-io: https://next-dojo-io-b2ayrh0r8.now.sh
* zh-cn-next-dojo-io: https://zh-cn-next-dojo-io-5euzcan5l.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* next-dojo-io: https://next-dojo-io-fhptzqd4v.now.sh
* zh-cn-next-dojo-io: https://zh-cn-next-dojo-io-rlkf0lwth.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* next-dojo-io: https://next-dojo-io-h52yinlve.now.sh
* zh-cn-next-dojo-io: https://zh-cn-next-dojo-io-30cz8nnk8.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* next-dojo-io: https://next-dojo-io-3hn340az9.now.sh
* zh-cn-next-dojo-io: https://zh-cn-next-dojo-io-15pmn60k1.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* next-dojo-io: https://next-dojo-io-f8s0we1va.now.sh
* zh-cn-next-dojo-io: https://zh-cn-next-dojo-io-7bnrf0u9l.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* next-dojo-io: https://next-dojo-io-agvvav1mv.now.sh
* zh-cn-next-dojo-io: https://zh-cn-next-dojo-io-2u1g4pene.now.sh

File: content/roadmap/en/dojo-7.0.0-release.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+* Live binding to function-based widget properties and children ([#564](https://github.com/framework/pull/564), [#665](https://github.com/framework/pull/665))
+* Check attached `node` is in the body ([#579](https://github.com/framework/pull/579))
+* Provide helper middleware `api` type ([#580](https://github.com/framework/pull/580))
+* Enable cache eviction with the `icache` ([#605](https://github.com/framework/pull/605))
+* Inert shim ([#619](https://github.com/framework/pull/619))
+* Support multiple CSS transition classes ([#628](https://github.com/framework/pull/628))
+* Inert middleware ([#623](https://github.com/framework/pull/623))
+* `focus` middleware testing mock ([#635](https://github.com/framework/pull/635))
+* `validity` middleware testing mock ([#631](https://github.com/framework/pull/631))
+* Type nls keys from the `i18n` format function ([#610](https://github.com/framework/pull/610))
```
Comment (dylans) [inline content/roadmap/en/dojo-7.0.0-release.md:29]:
```suggestion
* Type `nls` keys from the `i18n` format function ([#610](https://github.com/framework/pull/610))
```

File: content/roadmap/en/dojo-7.0.0-release.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+* Support passing `null` to the `domNode` on `.mount(` ([#652](https://github.com/framework/pull/652))
+* Influence property values with `diffProperties` middleware ([#653](https://github.com/framework/pull/653))
+* Support specifying an additional property to be considered part of the widget key ([#661](https://github.com/framework/pull/661))
+* Support automatic and lazy loading of CLDR data in i18n ([#657](https://github.com/framework/pull/657), [#677](https://github.com/framework/pull/677), [#686](https://github.com/framework/pull/686), [#693](https://github.com/framework/pull/693), [#694](https://github.com/framework/pull/694), [#724](https://github.com/framework/pull/724))
```
Comment (dylans) [inline content/roadmap/en/dojo-7.0.0-release.md:33]:
```suggestion
* Support automatic and lazy loading of CLDR data in `i18n` ([#657](https://github.com/framework/pull/657), [#677](https://github.com/framework/pull/677), [#686](https://github.com/framework/pull/686), [#693](https://github.com/framework/pull/693), [#694](https://github.com/framework/pull/694), [#724](https://github.com/framework/pull/724))
```

File: content/roadmap/en/dojo-7.0.0-release.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+* Unmount an application ([#680](https://github.com/framework/pull/680))
+* Dojo Resources ([#672](https://github.com/framework/pull/672), [#778](https://github.com/dojo/framework/pull/778))
+* Support setting and changing theme variants at runtime ([#684](https://github.com/framework/pull/684), [#711](https://github.com/framework/pull/711), [#720](https://github.com/framework/pull/720))
+* "slot" support for Dojo custom elements ([#714](https://github.com/framework/pull/714), [#721](https://github.com/framework/pull/721))
+* Object and array attribute support for custom elements [#756](https://github.com/dojo/framework/pull/756)
+* Forwards compatibility for TypeScript 3.8 ([#719](https://github.com/framework/pull/719))
```
Comment (dylans) [inline content/roadmap/en/dojo-7.0.0-release.md:39]:
```suggestion
* Forwards-compatibility for TypeScript 3.8 ([#719](https://github.com/framework/pull/719))
```

File: content/roadmap/en/dojo-7.0.0-release.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+* Configure document title on route change ([#725](https://github.com/framework/pull/725))
+* Skip setting the route when working with build time rendering and using the development server ([#729](https://github.com/framework/pull/729))
+* On-demand Build Time Rendering ([#246](https://github.com/dojo/webpack-contrib/pull/246))
+* Set document title for build time rendered pages ([#260](https://github.com/dojo/webpack-contrib/pull/260))
+* `.dojorc` schema validation for `@dojo/cli-build-app` ([#324](https://github.com/dojo/cli-build-app/pull/324))
+* asdasdasda ([#729](https://github.com/framework/pull/729))
```
Comment (dylans) [inline content/roadmap/en/dojo-7.0.0-release.md:45]: what is this?

File: content/roadmap/en/dojo-7.0.0-release.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+### Bug Fixes
+
+* Run key checks synchronously ([#547](https://github.com/framework/pull/547))
+* Ensure initial properties flag is set `false` ([#539](https://github.com/framework/pull/539))
+* Call `diffProperties` immediately when first registered ([#572](https://github.com/framework/pull/572))
+* Fix setting state in `focus` middleware ([#575](https://github.com/framework/pull/575))
+* Do not invalidate when check if a widget should focus ([#583](https://github.com/framework/pull/583))
```
Comment (dylans) [inline content/roadmap/en/dojo-7.0.0-release.md:53]:
```suggestion
* Do not invalidate when checking if a widget should focus ([#583](https://github.com/framework/pull/583))
```

File: content/roadmap/en/dojo-7.0.0-release.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+* Make assertion templates immutable ([#587](https://github.com/framework/pull/587))
+* Ensure mixin properties are inferred correctly when using TSX ([#594](https://github.com/framework/pull/594))
+* Run comparators when using harness.expect with an actual render output ([#602](https://github.com/framework/pull/602))
```
Comment (dylans) [inline content/roadmap/en/dojo-7.0.0-release.md:56]:
```suggestion
* Run comparators when using `harness.expect` with an actual render output ([#602](https://github.com/framework/pull/602))
```

File: content/roadmap/en/dojo-7.0.0-release.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+* Allow unsupported nodes to be ignore from harness's assertRender ([#599](https://github.com/framework/pull/599))
```
Comment (dylans) [inline content/roadmap/en/dojo-7.0.0-release.md:57]:
```suggestion
* Allow unsupported nodes to be ignored from harness's `assertRender` ([#599](https://github.com/framework/pull/599))
```

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* next-dojo-io: https://next-dojo-io-cut0dl3xz.vercel.app
* zh-cn-next-dojo-io: https://zh-cn-next-dojo-io-jqnpk01uy.vercel.app

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* next-dojo-io: https://next-dojo-io-65i1bezt2.vercel.app
* zh-cn-next-dojo-io: https://zh-cn-next-dojo-io-da0lvv0yv.vercel.app

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* next-dojo-io: https://next-dojo-io-6hn6futyj.vercel.app
* zh-cn-next-dojo-io: https://zh-cn-next-dojo-io-n9il91wj7.vercel.app

**Pull request merged**